mantidproject/mantid
# Add hidden column

This PR adds a hidden column as the last column of the DataProcessorWidget. The column allows the developer to add non-use-editable data on a row-by-row basis. Currently the developer can only add data for all rows. 

Fixes #20133 

**To test:**

This is again hard to test for a GUI which does not make use of the this feature. ISIS Reflectometry is currently the only GUI which makes use of this. In principle this should not have any effect on the Reflectometry reduction. Things to check are:

* Does the GUI look the same
* Manual testing can be done like this:
  - Login to the ICAT catalog
  - Open the ISIS Reflectometry interface change instrument to `POLREF`
  - Enter RB number `1520249` and transfer all runs
  - Run the reduction.
  - Compare some of the resulting reduced workspaces with the equivalient output from a version of Mantid which does not contain these changes.


**Release Notes** 
*Does not need to be in the release notes.*
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
